app/src/main/java/bubbleheads/buy_cook/MainActivity.java

```diff
@@ -26,6 +25,14 @@ protected void onCreate(final Bundle savedInstanceState) {
         return recipeList.getRecipes();
     }
 
+    public void setDetailedRecipe(final Recipe recipe) {
+        detailedRecipe = recipe;
+    }
+
+    public Recipe getChosenRecipe() {
```

Comment:
inconsistent names for method...setter is called setDETAILEDRecipe but getter is called getCHOSENRecipe...it should either be detailed, either chosen